content/kotsadm/installing/installing-embedded-cluster.md

```diff
@@ -32,11 +32,20 @@ tar xvf supergoodtool.tar
 cat install.sh | sudo bash -s airgap
 ```
 
-Note that the airgapped installer is not the same as an [airgapped application package](/kotsadm/installing/airgap-packages/).
-A KOTS application may be installed in airgapped mode on clusters installed online and vice versa.
-
 kURL currently uses `.tar.gz` extension for a `.tar` file, hence the `-o *.tar`.
 
+Note that the airgapped installer in the above step is not the application installer.  After this command completes, the application can be installed using the application airgap bundle.
+
+```bash
+kubectl kots install myapp \
+  --airgap-bundle ./myapp-1.0.165.airgap \
+  --license-file ./license.yaml \
+  --config-values ./config.yaml \
+  --namespace default \
+  --shared-password password
+```
+
+Applications can also be installed into existing clusters as described in [this doc](/kotsadm/installing/airgap-packages/).
```

Comment:
That's not a "doc". I don't like using "here", but it's slightly better.

```suggestion
Applications can also be installed into existing clusters as described [here](/kotsadm/installing/airgap-packages/).
```

Reply:
Why do we even need to link out to this? This whole page is about embedded clusters. If we want to redirect folks looking for existing cluster installs, shouldn't that be at the top of the doc?